views/components/Sidebar/index.jsx

```diff
@@ -6,7 +6,17 @@ import SidebarFooter from './SidebarFooter'
 
 const Sidebar = ({ user, sidebarOpen, onLogout }) => {
   return (
-    <Drawer variant="permanent" open={sidebarOpen}>
+    <Drawer
+      variant="permanent"
+      open={sidebarOpen}
+      sx={{
+        flexShrink: 0,
+        boxSizing: 'border-box',
+        width: '248px',
+        height: '100vh',
```

Comment:
do we have to specify the height for this premade component..?

Reply:
yeah we need it in this case or the drawer won't be the full height of the screen.